post/2017/02-19-a-breeze-of-2017.md

```diff
@@ -0,0 +1,73 @@
+---
+layout:     post
+title:      A breeze of 2017
+date:       2017-02-19 19:25:00 +0200
+author:     Nooby
+summary:    Some of our plans for 2017.
+---
+
+# A breeze of 2017
+It has been [almost a year](https://inexor.org/blog/2016/03-19-doxylamine-alpha-edition) since we have updated this blog. This has been for various reasons, foremost it was not of priority to the team as we did concentrate on the game itself.
+This post will start to explain a few changes we did implement last year and continues with some issues we are working on right now.
+
+
+## What happened with Inexor?
+We have kept modifying Inexor to be more modable and flexible, which in turn leaded to the design of e.g. a synchronizable tree structure of the game.
+It is a rather complex process, which lead to several new code modules like:
+
+  * an [initial process](https://github.com/inexor-game/code/wiki/Inexor-Tree-API) where we can bind CubeScript variables via RPC
```

Comment:
That doesn't fit. From an one year perspective Flex and Tree are completely new.